ci_scripts/ci_post_clone.sh

```diff
@@ -68,9 +31,48 @@ APP_ID_IOS = org.mozilla.ios.FirefoxVPN
 NETEXT_ID_IOS = org.mozilla.ios.FirefoxVPN.network-extension
 EOF
 
-if [ $CI_PRODUCT_PLATFORM == 'macOS' ]
-then
-  ./scripts/macos/apple_compile.sh macos
-else
-  ./scripts/macos/apple_compile.sh ios -q /Volumes/workspace/repository/qt_ios/$QTVERSION/macos/bin
-fi
+# TODO: REVERT HARDCODED LINK HERE
+curl -L https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/Je1neDwuS6KKfuNvUrLH6g/runs/0/artifacts/public%2Fbuild%2Fqt6_ios.zip --output qt_ios.zip
+unzip qt_ios.zip
+ls
+
+QTVERSION=$(ls qt_ios)
+echo "Using QT:$QTVERSION"
+
+export QT_IOS_BIN=/Volumes/workspace/repository/qt_ios/$QTVERSION/ios/bin
+export PATH=/Volumes/workspace/repository/qt_ios/$QTVERSION/ios/bin:/Volumes/workspace/repository/qt_ios/$QTVERSION/macos/bin:$PATH
+
+export LC_ALL=en_US.utf-8
+export LANG=en_US.utf-8
+
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source "$HOME/.cargo/env"
+rustup target add aarch64-apple-ios
+
+# Install go
+brew install go
+
+# Install Python deps
+python3 -m pip install -r requirements.txt --user
+export PYTHONIOENCODING="UTF-8"
+
+# Install cmake
+brew install cmake
+
+# In previous iterations "Mozilla VPN" didn't use to have a space in it,
+# to avoid the pains of migrations we will just rename here. 
+# This is harmless, but still very hacky.
+WORKSPACE_ROOT="$( cd "$(dirname "$0")/.." ; pwd -P )"
+sed -i.bak -E \
+    -e "s/Mozilla VPN/MozillaVPN/" \
+    "${WORKSPACE_ROOT}/CMakeLists.txt"
+sed -i.bak -E \
+    -e "s/MAIN_TARGET mozillavpn/MAIN_TARGET MozillaVPN/" \
+    "${WORKSPACE_ROOT}/CMakeLists.txt"
```

Comment:
@bakulf this is the reason for replacing `mozillavpn` with the variable. I wanted to make this last replace easier.

Reply:
This can be achieved  using cmake flags.

Reply:
I have attempted to achieve this by changing multiple variables listed in https://cmake.org/cmake/help/latest/manual/cmake-variables.7.html and https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#target-properties without success in either changing the project name or the mozillavpn target name. 

The objective here is to generate an `.xcodeproj` file with the name `MozillaVPN.xcodeproj` and the main scheme to be also `MozillaVPN`. @oskirby @bakulf please let me know if you know how to do this.